pandas/plotting/_converter.py

```diff
@@ -178,6 +180,16 @@ class DatetimeConverter(dates.DateConverter):
 
     @staticmethod
     def convert(values, unit, axis):
+        # values might be a 1-d array, or a list-like of arrays.
+        if is_list_like(values) and is_list_like(values[0]):
```

Comment:
ahh could add that to the condition as well (so even more reason to make this an instrospection function).